### PR TITLE
Add `[other].SHOW_FOOTER_POWERED_BY` setting to hide `Powered by`

### PR DESCRIPTION
**Pull request by Yakov5776**
This allows you to hide the "Powered by" text in footer via `SHOW_FOOTER_POWERED_BY` flag in configuration.

### CODE REVIEW

**Comment by Yakov5776** [comment]
tested, it works as intended.

File: custom/conf/app.example.ini [hunk 1]
```diff
@@ -2315,6 +2315,8 @@ LEVEL = Info
 ;SHOW_FOOTER_VERSION = true
 ;; Show template execution time in the footer
 ;SHOW_FOOTER_TEMPLATE_LOAD_TIME = true
+;; Show powered by text in the footer
```
Comment (silverwind) [inline custom/conf/app.example.ini]:
```suggestion
;; Show the "powered by" text in the footer
```

File: docs/content/administration/config-cheat-sheet.en-us.md [hunk 2]
```diff
@@ -1429,5 +1429,6 @@ Like `uses: https://gitea.com/actions/checkout@v4` or `uses: http://your-git-ser
 
 - `SHOW_FOOTER_VERSION`: **true**: Show Gitea and Go version information in the footer.
 - `SHOW_FOOTER_TEMPLATE_LOAD_TIME`: **true**: Show time of template execution in the footer.
+- `SHOW_FOOTER_POWERED_BY`: **true**: Show the powered by text in the footer.
```
Comment (silverwind) [inline docs/content/administration/config-cheat-sheet.en-us.md]:
```suggestion
- `SHOW_FOOTER_POWERED_BY`: **true**: Show the "powered by" text in the footer.
```

File: docs/content/administration/config-cheat-sheet.zh-cn.md [hunk 3]
```diff
@@ -1353,5 +1353,6 @@ PROXY_HOSTS = *.github.com
 
 - `SHOW_FOOTER_VERSION`: **true**: 在页面底部显示Gitea的版本。
 - `SHOW_FOOTER_TEMPLATE_LOAD_TIME`: **true**: 在页脚显示模板执行的时间。
+- `SHOW_FOOTER_POWERED_BY`: **true**: 在页脚显示“由...提供动力”的文本。
```
Comment (silverwind) [inline docs/content/administration/config-cheat-sheet.zh-cn.md:1356]:
```suggestion
- `SHOW_FOOTER_POWERED_BY`: **true**: 在页脚中显示“供电”文本

```

**Review by silverwind — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

**Pull request merged**